test/e2e/seeder/anvil.ts

```diff
@@ -0,0 +1,125 @@
+import { createAnvil, Anvil as AnvilType } from '@viem/anvil';
+import { createAnvilClients } from './anvil-clients';
+
+type Hardfork =
+  | 'Frontier'
+  | 'Homestead'
+  | 'Dao'
+  | 'Tangerine'
+  | 'SpuriousDragon'
+  | 'Byzantium'
+  | 'Constantinople'
+  | 'Petersburg'
+  | 'Istanbul'
+  | 'Muirglacier'
+  | 'Berlin'
+  | 'London'
+  | 'ArrowGlacier'
+  | 'GrayGlacier'
+  | 'Paris'
+  | 'Shanghai'
+  | 'Latest';
+
+const defaultOptions = {
+  balance: 25,
+  blockTime: 2,
+  chainId: 1337,
+  gasLimit: 30000000,
+  gasPrice: 2000000000,
+  hardfork: 'Muirglacier' as Hardfork,
+  host: '127.0.0.1',
+  mnemonic:
+    'spread raise short crane omit tent fringe mandate neglect detail suspect cradle',
+  port: 8545,
+};
```

Comment:
This are the default network options. Since we have them defined here, there's no need for `defaultAnvilOptions`, so we don't need to pass anything in the spec files, when we want the default options (see spec files above).

Likewise, we have ganache options defined in the ganache file, we wouldn't need the `defaultGanacheOptions` (but it's implemented like this)